src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionCategoryController.java

```diff
@@ -28,10 +29,10 @@
 public class FinancialTransactionCategoryController{
 
     @GetMapping("/{id}")
-    ResponseEntity<FinancialTransactionCategoryDTO> getFinancialTransactionCategoryById(@Min(1) @NotNull @PathVariable Long id){
+    ResponseEntity<FinancialTransactionCategoryCounterDTO> getFinancialTransactionCategoryById(@Min(1) @NotNull @PathVariable Long id){
         //TODO Necessary code implementation
-        FinancialTransactionCategoryDTO financialTransactionCategoryDTO = null;
-        return new ResponseEntity<>(financialTransactionCategoryDTO, HttpStatus.OK);
+        FinancialTransactionCategoryCounterDTO financialTransactionCategoryCounterDTO = null;
```

Comment:
This name seems to be very confusing, based on that I would think that it contains counter and nothing more. Also now comparing to getting all categories endpoint it has only counter as additional property and in future it can have more properties.
Proposals:
- `FinancialTransactionExtendedCategory`
- `FinancialTransactionFullCategory`
- `ExtendedFinancialTransactionCategory`
Or call this one `FinancialTransactionCategory` and for getting all transactions -> `FinancialTransactionCategoryListItem`

Reply:
FinancialTransaction**Detailed**CategoryDTO ?

Reply:
Also better than `FinancialTransactionCategoryCounterDTO`, fine for me